wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
-    -   **1 grid to the right** It is not always obvious **why** a hit object should be moved a grid up/right/down/left. It is fine to suggest this, but explain **why** it should be moved (typically it's to make a pattern symmetrical). This sort of suggestion, however, does not offer much to improve a beatmap. Keep in mind a mod post should not only consist of such suggestions.
-    -   **This shape is ugly** Whether the shape is made of [hit circles](Hit_circle "wikilink"), [sliders](sliders "wikilink") or both, the mapper obviously will not understand what you are asking for. What looks aesthetically pleasing to the mapper may not always look aesthetically pleasing to you. Please understand that the osu! community comes from across the world and respect each player's opinions. Instead of stating the slider is outright ugly, it is better to say "This slider looks weird in my opinion" and then to offer an alternative (via an in-post attachment or an external link).
+#### Difficulty
 
-2.  **Offensive Language** Even if the mapper is a close friend, please try to keep the mod post clean. It is understandable that certain patterns may seem completely illogical to you, but that is not a reason for insulting the mapper. Refraining from terms such as "WTF is this?" or "This is completely stupid" can help to repress any potential argument and create a better osu! environment.
+The difficulty settings can be found in the song setup dialog.
+These are important to game play so that the song is at a fair level of difficulty for the varying difficulties a mapset can have.
+
+For example, an [Easy](/wiki/Easy) difficulty does not really need an [approach rate](/wiki/approach_rate) (AR) of 7 or an [HP drain rate](/wiki/HP_drain_rate) (HP) of 8.
+Using these values for an Easy difficulty makes this completely illogical and will make the difficulty much harder than it should be.
+
+### Background, Skin, and Combo Colors
+
+All beatmaps must have a background image.
+The skin you supply should not blend with the background image or greatly hinder (or chellenge) game play.
+For example, complicated hit circle design, huge hit bursts, etc.
+
+There are also many [skinning](/wiki/skinning) rules that should be followed.
+Similar to skinning, combo colors should not blend in with the background either.
+These combo colors should also be very distinct and clear.
+The mapper should not have two almost identical combo colors unless they are separated by a very contrasting color in between.
+
+### Storyboarding
+
+_See also: [Storyboarding](/wiki/Storyboarding)._
+
+If you spot anything weird while the storyboard is playing (e.g. bad rendering of pictures, emptiness, etc.), it is a good idea to tell the [storyboarder](/wiki/storyboarder) so that they can fix it.
+The storyboard generally should not exceed an [SB load](/wiki/SB_load) of `5.0x`.
```

Comment:
This is likely going to be scrapped from the RC, just saying: might want to refer to https://osu.ppy.sh/forum/t/559005 for the time being